R/rgba_int_to_raster.R

```diff
@@ -0,0 +1,18 @@
+#' rgba_int_to_raster
+#'
+#' Create raster from given RGBA matrix.
+#'
+#' @param i32RGBA Integer RGBA matrix (`red`, `green`, `blue` and `alpha` channels, dimension nxmx4, values ~ 0-255).
+#'
+#' @return Raster result.
+#'
+#' @export
+#' @useDynLib scattermore, .registration=TRUE
+#' @importFrom grDevices as.raster
```

Comment:
this should have an entry in `Imports:` in DESCRIPTION (see the original scattermore)